src/DataViews/include/DataViews/DataView.h

```diff
@@ -5,7 +5,7 @@
 #ifndef DATA_VIEWS_DATA_VIEW_H_
 #define DATA_VIEWS_DATA_VIEW_H_
 
-#include <absl/container/flat_hash_set.h>
+#include <absl/container/flat_hash_map.h>
```

Comment:
This change is probably not needed anymore, right? (If it still is, please also add an include for flat_hash_set. It's used as a member variable down below.